tests/brun/substr_r-1.txt

```diff
@@ -0,0 +1,3 @@
+brun -c '(substr (q . "abcdefghijkl") (q . 9) (q . 10))'
```

Comment:
this looks like a duplicate test, taking two indices

Reply:
I removed it, and another one that is 3-arg.

Reply:
Also renamed/renumbered all the `substr` tests in `brun`.